frank-flow/src/frontend/src/app/header/header.component.html

```diff
@@ -25,6 +25,13 @@
     >
       <fa-icon icon="pen"></fa-icon>
     </app-button>
+    <app-button
+      (click)="openConfigurationSettings()"
+      color="dark"
+      title="Modify configuration settings"
+    >
+      <fa-icon icon="cogs"></fa-icon>
+    </app-button>
```

Comment:
In the feature, it would be nice to disable the button if the configuration contains errors. But there is an issue for this (#425).

It might be a good idea to hide the button entirely if the currentFile.type is not FileType.configuration.

Reply:
added a check to hide and show the button